starters/gatsby-starter-minimal-ts/gatsby-config.ts

```diff
@@ -1,9 +1,7 @@
 /** @type {import('gatsby').GatsbyConfig} */
 module.exports = {
   siteMetadata: {
-      siteUrl: `https://www.yourdomain.tld`,
+    siteUrl: `https://www.yourdomain.tld`,
```

Comment:
Needs to be a JS file :)

Reply:
😅 It's a sign I need to log off for the day